Assignment2/Sixtrax-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,69 @@
+{
+	"@context":{
+			"class":"http://www.example.org/Class#",
+      		"element":"http://www.example.org/Elements#",
+			"pr":"http://www.example.org/Properties#",
+      		"msr":"http://www.example.org/Measurements#",
+      		"user":"http://www.example.org/Users#",
+			"xsd":"http://www.w3.org/2001/XMLSchema#",
+			"Class01":"class:Class01",
+      		"includes":"pr:includes",
+			"hasMeasurement":"pr:hasMeasurement",
+			"hasOwner":"pr:hasOwner",
+			"hasTemperature":"pr:hasTemperature",
+			"atTime":"pr:atTime",
+			"hasName":"pr:hasName",
+			"Sensor029":"element:Sensor029",
+			"Computer101":"element:Computer101",
+			"mMeasurement8401":"msr:Measurement8401",
+			"User10A":"user:User10A"
+	},
+	"Class01":{
+		"@id":"http://www.example.org/Class01",
+		"@type":"class:Class01",
+		"includes":[{
+			"@id":"element:Sensor029/",
+			"@type":"element"
+		},
+		{
+			"@id":"element:Computer101/",
+			"@type":"element"
+		}]
+		"Sensor029":{

```

Comment:
You don't need to define "Sensor029" because it isn't an attribute, it is a resource, thus I suggest you remove this definition. You can complete its definition by adding the "hasMeasurement" definition inside the Sensor resource that you already defined inside the "includes" array.  
This comment also applies for "Computer101", "Measurement8401", and "User10A".